recipes/nbpresent/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set version = "3.0.1" %}
+{% set gh_org = "Anaconda-Platform" %}
+{% set gh_repo = "nbpresent" %}
+{% set commit = "a33eeb1783741f15430ca03286eb65382d112eb0" %}
+
+package:
+  name: {{ gh_repo }}
+  version: {{ version }}
+
+source:
+  ## bring back when not [WIP]
```

Comment:
Yes, yes... was just enjoying a frosted barley pop after getting this terrible bastard running. Hope to take off [wip] soon :)